.github/ruff.toml

```diff
@@ -0,0 +1,6 @@
+select = ["E"]
+ignore = ["E402", "E722"]
```

Comment:
please add some comments here describing what these codes are, so we don't have to look them up